core/core_test.go

```diff
@@ -520,33 +520,72 @@ func Test_ideaExitCode(t *testing.T) {
 func TestSetupLicense(t *testing.T) {
 	Prod.Code = "QDJVM"
 	Prod.EAP = false
-	license := `{"licenseId":"VA5HGQWQH6","licenseKey":"VA5HGQWQH6","expirationDate":"2023-07-31","licensePlan":"EAP_ULTIMATE_PLUS","projectIdHash":"hash"}`
-	expectedKey := "VA5HGQWQH6"
-	expectedHash := "hash"
+	for _, tc := range []struct {
+		name            string
+		license         string
+		expectedKey     string
+		expectedHash    string
+		expectedOrgHash string
+	}{
+		{
+			name:            "valid license",
+			license:         `{"licenseId":"VA5HGQWQH6","licenseKey":"VA5HGQWQH6","expirationDate":"2023-07-31","licensePlan":"EAP_ULTIMATE_PLUS","projectIdHash":"hash","organizationIdHash":"org hash"}`,
+			expectedKey:     "VA5HGQWQH6",
+			expectedHash:    "hash",
+			expectedOrgHash: "org hash",
+		},
+		{
+			name:            "no organizationIdHash",
+			license:         `{"licenseId":"VA5HGQWQH6","licenseKey":"VA5HGQWQH6","expirationDate":"2023-07-31","licensePlan":"EAP_ULTIMATE_PLUS","projectIdHash":"hash"}`,
+			expectedKey:     "VA5HGQWQH6",
+			expectedHash:    "hash",
+			expectedOrgHash: "",
```

Comment:
Why empty value? there should be no json field at all?

Reply:
Ok, missed that its expected

Reply:
That's a go way, string cannot be nil. Logic and test written in a way to not set env variable in a case of empty string.